stdpopsim/catalog/AnoCar/genome_data.py

```diff
@@ -1,21 +1,20 @@
 # File autogenerated from Ensembl REST API. Do not edit.
 data = {
-    "assembly_accession": "GCA_000090745.1",
-    "assembly_name": "AnoCar2.0",
+    "assembly_accession": "GCA_000090745.2",
+    "assembly_name": "AnoCar2.0v2",
     "chromosomes": {
-        "1": {"length": 263920458, "synonyms": []},
-        "2": {"length": 199619895, "synonyms": []},
-        "3": {"length": 204416410, "synonyms": []},
-        "4": {"length": 156502444, "synonyms": []},
-        "5": {"length": 150641573, "synonyms": []},
-        "6": {"length": 80741955, "synonyms": []},
-        "LGa": {"length": 7025928, "synonyms": []},
-        "LGb": {"length": 3271537, "synonyms": []},
-        "LGc": {"length": 9478905, "synonyms": []},
-        "LGd": {"length": 1094478, "synonyms": []},
-        "LGf": {"length": 4257874, "synonyms": []},
-        "LGg": {"length": 424765, "synonyms": []},
-        "LGh": {"length": 248369, "synonyms": []},
-        "MT": {"length": 17223, "synonyms": []},
+        "1": {"length": 263920458, "synonyms": ["chr1"]},
+        "2": {"length": 199619895, "synonyms": ["chr2"]},
+        "3": {"length": 204416410, "synonyms": ["chr3"]},
+        "4": {"length": 156502444, "synonyms": ["chr4"]},
+        "5": {"length": 150641573, "synonyms": ["chr5"]},
+        "6": {"length": 80741955, "synonyms": ["chr6"]},
+        "a": {"length": 7025928, "synonyms": ["chrLGa"]},
+        "b": {"length": 3271537, "synonyms": ["chrLGb"]},
+        "c": {"length": 9478905, "synonyms": ["chrLGc"]},
+        "d": {"length": 1094478, "synonyms": ["chrLGd"]},
+        "f": {"length": 4257874, "synonyms": ["chrLGf"]},
+        "g": {"length": 424765, "synonyms": ["chrLGg"]},
+        "h": {"length": 248369, "synonyms": ["chrLGh"]},
```

Comment:
wtf, the previous chromosome names (e.g., `LGa`) are not included in the synonyms. I vote to leave well enough alone there. But - why no mitochondria?

Reply:
And gee, the mitochondria shows on ensembl:
![Screenshot from 2023-12-12 09-15-25](https://github.com/popsim-consortium/stdpopsim/assets/1046249/564c1940-a787-4dfe-ad5f-18802092811f)

Reply:
ah I see what happened here: consulting http://rest.ensembl.org/info/assembly/anolis_carolinensis they've removed the mitochondria from the "karyotype" list of chromosomes, but it's still there, at the end of a list of a bajillion tiny contigs:
```
  - 
    coord_system: primary_assembly
    length: 17223
    name: MT
```